### ISSUE TITLE
Incorrect eslint error on navigator

### ISSUE THREAD

**Issue by chualuoi** [amilajack/eslint-plugin-compat#380]
## Step to reproduce
1. git clone https://github.com/chualuoi/eslint-plugin-compat-demo
2. cd eslint-plugin-compat-demo
3. npm install && npm run lint

## Expected
No linting error

## Actual
```
navigator is not supported in Firefox 55
```

**Comment by mnicpt** [comment]
Same here.

**Comment by Semigradsky** [comment]
Looks like it should be fixed automatically after https://github.com/amilajack/ast-metadata-inferer/issues/129

**Comment by Semigradsky** [comment]
It was fixed in `eslint-plugin-compat`@3.11.1 👍
